### PR TITLE
Migrating the tidy pack to demisto registry

### PR DESCRIPTION
**Pull request by hod-alpert**
Migrating the tidy pack to demisto registry

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -1993,3 +1993,31 @@ update_configs:
...
+  - package_manager: docker
+    directory: /docker/tidy
+    update_schedule: daily
+    automerged_updates:
+    - match:
+        dependency_name: demisto/*
+        update_type: all
+
+  - package_manager: python
+    directory: /docker/tidy123
```
Comment (glicht) [inline .dependabot/config.yml:2011]: What is this one? I think this was added by mistake.

Comment (hod-alpert) [reply .dependabot/config.yml:2011]: Yes,  accident.

Removed it

File: .dependabot/config.yml [hunk 2]
```diff
@@ -1993,3 +1993,25 @@ update_configs:
...
+  - package_manager: docker
+    directory: /docker/tidy
+    update_schedule: daily
+    automerged_updates:
+    - match:
+        dependency_name: demisto/*
+        update_type: all
+
+  - package_manager: docker
+    directory: /docker/tidy123
```
Comment (glicht) [inline .dependabot/config.yml:2011]: you still have here tidy123

**Comment by glicht** [comment]
You can also exclude `ducutils` for the licensing as we do for ansible-runner. See: https://github.com/demisto/dockerfiles/blob/master/docker/packages_license_check_exclude.json#L8

**Review by glicht — APPROVED** [review]

**Review by glicht — APPROVED** [review]

**Pull request merged**